src/solver/simulation/include/antares/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -608,5 +608,8 @@ struct PROBLEME_HEBDO
     std::unique_ptr<PROBLEME_ANTARES_A_RESOUDRE> ProblemeAResoudre;
 
     double maxPminThermiqueByDay[366];
+
+    Study::SystemModel::System* modelerSystem_;                               // for hybrid studies
+    Optimisation::LinearProblemApi::ILinearProblemData* linear_problem_data_; // for hybrid studies
```

Comment:
There are multiple instances of `PROBLEME_HEBDO`, but only a single instance of `Study`. This is a source of errors & undefined behavior (race conditions).

For now it's OK, but when running hydrid studies in parallel we'll probably want different instances here.

Please add a `const` qualifier if possible, and also a TODO note.